src/app/shared/components/addon-list/edit-addon-dialog/edit-addon-dialog.component.ts

```diff
@@ -55,10 +56,16 @@ export class EditAddonDialogComponent implements OnInit {
     const variables = {};
 
     Object.keys(this.form.controls).forEach(key => {
+      if (key === 'continuouslyReconcile') {
```

Comment:
I'm not a fan of these kind of exceptions. IMO it would be better to put this checkbox outside this form. In a second form or as a standalone control.